packages/animations/test/fade_transition_test.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:animations/src/fade_transition.dart';
+import 'package:animations/src/modal.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/widgets.dart';
+
+void main() {
```

Comment:
This will probably need a few more tests?

Reply:
Yup! It's still a draft PR so it's not quite there yet -- I'll need to write tests for `modal.dart` and address the earlier code review comments